core/src/test/java/com/linecorp/armeria/server/RouteDecoratingTest.java

```diff
@@ -187,7 +187,7 @@ void decorateInOrder(String path, List<Integer> orders) {
             "/api/admin/1, , 401, ",
             "/assets/index.html, , 200, ",
             "/assets/resources/index.html, , 200, public",
-            "/assets/resources/private/profile.jpg, , 200, private",
+            "/assets/resources/private/profile.jpg, , 200, public",
```

Comment:
Shouldn't we update the decorator ordering instead of updating the test parameters? Returning `public` on `.../private/...` doesn't sound right to me.

Reply:
Yes, I fixed the order. 👍